### DISCUSSION TITLE
Need an email id for me to resend the id card and university offer letter ; my laptop camera is unable to show the clear picture

### DISCUSSION THREAD

**Discussion by coursegit24** [community/community#139416] [GitHub Education]
### Hi everyone,

Need an email id for me to resend the id card and university offer letter ; my laptop camera is unable to show the clear picture

**Comment by davevad93** [comment]
Hi @coursegit24 ,

If possible, try to submit the request using a smartphone, the camera is better than the webcam for taking pictures.
